tests/test_utils.py

```diff
@@ -63,3 +64,12 @@ def test_make_log_dirs():
 			pass
 	finally:
 		shutil.rmtree(filepath)
+
+@pytest.mark.parametrize("mpi_size", [1, 4, 10, 100, 1000, 515, 14365, 13512, 86400])
+@pytest.mark.parametrize("num_live_points_missing", [0, 1, 4, 10, 100, 1000, 515, 14365, 13512, 86400])
```

Comment:
This looks good. I think >10000 is a bit extreme and unusual, for both parameters. Maybe keep one of these high numbers. You can add some primes between 10 and 1000.

I am presuming `make test` is very fast on this function.

One day I may use [hypothesis](https://hypothesis.readthedocs.io/en/latest/quickstart.html) to do such tests...

Reply:
I have tried hypothesis before. This would work here, but it is literally an integer division arithmetics.